files/pg_hba.conf

```diff
@@ -1,6 +0,0 @@
-# "local" is for Unix domain socket connections only
```

Comment:
is files/pg_hba.conf extraneous, since we'd be placing a template?

Reply:
Yes, I think it is extraneous.

Reply:
It seems that files/pg_hba.conf is already removed in my branch.